pyproject.toml

```diff
@@ -7,10 +7,13 @@ requires = [
 build-backend = "setuptools.build_meta"
 
 [project]
-    name = "idaes_ui"
+    ######
+    #temporary project name with "idaes-ui-test", when package please rename this with the final name!
+    ######
+    name = "idaes-ui-test"
     description = "IDAES Process Systems Engineering User Interface"
     readme = "README.md"
-    version = "0.0.1"
+    version = "0.0.14"
```

Comment:
I think this should be 1.0.0 (on real pypi) considering that IDAES-pse is at 2.x

Reply:
Can I make 2 versions and use them on different PyPI, this may be easier for future dev, when testing on PyPI test.
When ready to public to PyPI can use the second version, and when testing just use the first one.

```sh
# For PyPI test repo
version = "0.0.14"

# For PyPI
#version = "1.0.0"
```